Re-add relevant lines that were removed with upstream commit e6f429e64b1e to be
able to build a wheel containing the Python source files.

--- pyproject.toml.orig	2025-03-22 11:37:42 UTC
+++ pyproject.toml
@@ -50,6 +50,9 @@ getter = "scripts.get_version:get_version"
 getter = "scripts.get_version:get_version"
 
 [tool.pdm.build]
+package-dir = "src"
+editable-backend = "editables"
+includes = ["src/mkdocstrings_handlers"]
 # Include as much as possible in the source distribution, to help redistributors.
 excludes = ["**/.pytest_cache"]
 source-includes = [
